src/locales/ja-JP/stream.ftl

```diff
@@ -0,0 +1,895 @@
+  現在ひとつの会話を表示しています。
+comments-inReplyTo = <Username></Username> に返信する
+comments-replyingTo = <Username></Username> に返信する
+
+comments-reportButton-report = 報告
+comments-reportButton-reported = 報告済み
+comments-reportButton-aria-report =
+  .aria-label = {$username} のコメントを報告する
+comments-reportButton-aria-reported =
+  .aria-label = 報告済み
+
+comments-sortMenu-sortBy = 並び替え
+comments-sortMenu-newest = 最新
+comments-sortMenu-oldest = 最も古い
+comments-sortMenu-mostReplies = もっと返信を見る
+
+comments-userPopover =
+  .description = より多くのユーザー情報を含むポップオーバー
+comments-userPopover-memberSince = { DATETIME($timestamp, year: "numeric", month: "long", day: "numeric") } 以降のメンバー
+comments-userPopover-ignore = 無視する
+
+comments-userIgnorePopover-ignoreUser = {$username} を無視しますか？
+comments-userIgnorePopover-description =
+  When you ignore a commenter, all comments they wrote on the site will be hidden from you.
+ You can undo this later from My Profile.
+comments-userIgnorePopover-ignore = 無視する
+comments-userIgnorePopover-cancel = キャンセル
+
+comments-userBanPopover-title = {$username} を禁止しますか？
+comments-userBanPopover-description =
+  禁止されると、このユーザーはコメントしたり、リアクションを使用したり、コメントを報告したりできなくなります。
+  このコメントも拒否されます。
+comments-userBanPopover-cancel = キャンセル
+comments-userBanPopover-ban = 禁止する
+
+comments-moderationDropdown-popover =
+  .description = コメントをモデレートするためのポップオーバーメニュー
+comments-moderationDropdown-feature = 特集
+comments-moderationDropdown-unfeature = 特集しない
+comments-moderationDropdown-approve = 承認
+comments-moderationDropdown-approved = 承認済み
+comments-moderationDropdown-reject = 排除
+comments-moderationDropdown-rejected = 排除済み
+comments-moderationDropdown-ban = 禁止ユーザー
+comments-moderationDropdown-banned = 禁止済み
+comments-moderationDropdown-goToModerate =
+comments-moderationDropdown-moderationView = モデレーションビュー
+comments-moderationDropdown-moderateStory = モデレートストーリー
+comments-moderationDropdown-caretButton =
+  .aria-label = モデレート
+
+comments-moderationRejectedTombstone-title = このコメントは排除済みです
+comments-moderationRejectedTombstone-moderateLink =
+  この決定を確認するために、モデレーションに移動
+
+comments-featuredTag = 特集
+
+# $reaction could be "Respect" as an example. Be careful when translating to other languages with different grammar cases.
+comments-react =
+  .aria-label = {$count ->
+    [0] {$reaction} 投稿者 {$username}
+    *[other] {$reaction} 投稿者 {$username} (合計： {$count})
+  }
+
+# $reaction could be "Respected" as an example. Be careful when translating to other languages with different grammar cases.
+comments-reacted =
+  .aria-label = {$count ->
+    [0] {$reaction} 投稿者 {$username}
+    [one] {$reaction} 投稿者 {$username}
+    *[other] {$reaction} comment by {$username} (Total: {$count})
+  }
+
+comments-jumpToComment-title = あなたの返信は、下記に投稿されています
+comments-jumpToComment-GoToReply = 返信する
+
+comments-mobileToolbar-closeButton =
+  .aria-label = 閉じる
+comments-mobileToolbar-unmarkAll = 全てのマークを外す
+comments-mobileToolbar-nextUnread = 次の未読コメント
+
+comments-replyChangedWarning-theCommentHasJust =

```

Comment:
It looks like this translation is English. Did you mean to include a Japanese translation here?